docs/tutorial/getting-started-with-istio-mesh.md

```diff
@@ -0,0 +1,259 @@
+
+## Why Do We Need Charmed Istio?
+
+While the application works, it has critical security vulnerabilities:
+
+### No Encryption Between Services
+
+All microservice communication happens over plain HTTP. This means sensitive data is exposed to anyone with network access and can be intercepted or modified.
+
+While Kubernetes supports [manual TLS configuration](https://kubernetes.io/docs/tasks/tls/), managing certificates for each microservice becomes exponentially complex as your architecture scales - requiring certificate generation, distribution, rotation, and troubleshooting across potentially hundreds of services.
+
+### Unrestricted Service Communication
+
+Every service can access every other service without restrictions. If an attacker compromises one service, they immediately gain access to your entire application. You can verify this unrestricted access:
+
+```bash
+# All endpoints are accessible with any HTTP method
+kubectl exec -n bookinfo bookinfo-productpage-k8s-0 -- curl -s http://bookinfo-details-k8s.bookinfo.svc.cluster.local:9080/
+kubectl exec -n bookinfo bookinfo-productpage-k8s-0 -- curl -s http://bookinfo-details-k8s.bookinfo.svc.cluster.local:9080/health
+kubectl exec -n bookinfo bookinfo-productpage-k8s-0 -- curl -s http://bookinfo-details-k8s.bookinfo.svc.cluster.local:9080/details/1
+
+# Even potentially dangerous methods like POST work
+kubectl exec -n bookinfo bookinfo-productpage-k8s-0 -- curl -s -X POST http://bookinfo-details-k8s.bookinfo.svc.cluster.local:9080/details/1 -d '{}'
+```
+
+There are no [authorization policies](https://istio.io/latest/docs/reference/config/security/authorization-policy/) controlling:
+- Which services can communicate
+- What HTTP methods are allowed
+- Which endpoints can be accessed
+
+### The Charmed Istio Solution
+
+Charmed Istio automatically provides:
+- **mTLS encryption** between all services without manual certificate management
+- **Fine-grained authorization policies** defining exactly which services can communicate
+- **Simplified Day 2 operations** through Juju relations, making security configuration as easy as connecting charms
+
+Let's secure our application with Charmed Istio.
+
+## Secure with Service Mesh
+
+### Step 6: Add Services to the Mesh
+
+Deploy the `istio-beacon-k8s` charm and connect it to the bookinfo backend charms:
+
+```bash
+juju deploy istio-beacon-k8s --channel 2/edge
+juju integrate bookinfo-details-k8s istio-beacon-k8s
+juju integrate bookinfo-reviews-k8s istio-beacon-k8s
+```
+
+With the above two simple commands, the `istio-beacon-k8s` charm 
+
+- Adds the `bookinfo-details-k8s` and `bookinfo-reviews-k8s` charms to the Istio service mesh
+- Applies [mTLS](https://istio.io/latest/blog/2023/secure-apps-with-istio/) for the traffic between the services in the mesh
+- Applies a **deny-by-default** [authorization policy](https://istio.io/latest/docs/reference/config/security/authorization-policy/) 
```

Comment:
suggestion... not sure its perfect though.  I wanted to explain a tiny bit more about what deny-by-default meant in this context

```suggestion
- Applies a **deny-by-default** [authorization policy](https://istio.io/latest/docs/reference/config/security/authorization-policy/) which means traffic to anything on the mesh is denied unless explicitly allowed
```